samtranslator/model/sam_resources.py

```diff
@@ -1861,6 +1862,7 @@ def generate_resources(
             raise InvalidResourceException(
                 self.logical_id,
                 f"Unable to create connector from {source.resource_type} to {destination.resource_type}; it's not supported or the template is invalid.",
+                {UNSUPPORTED_CONNECTOR_PROFILE_TYPE: f"{source.resource_type} to {destination.resource_type}"},
```

Comment:
nit: I like structured type `{source.resource_type: destination.resource_type}` (a dict) more than a string.

Reply:
Nice, I like it.